### DISCUSSION TITLE
Some structural suggestions for the Nillion docs

### DISCUSSION THREAD

**Discussion by a3s7p** [NillionNetwork/discussions#126] [Ideas]
The docs are excellent in content but can be fairly hard to navigate for new users.

Suggestions:

- split docs in sections per "persona"/role of end user like on Discord: developer, researcher, user etc
  - some articles may belong to 2 or more sections at the same time, this is fine
- for each section, have a "reasonably expected" intro
  - e.g. researchers want the white papers, devs want the SDK, users want existing apps
- for SDKs, keep the body text unified while providing language-specific code sections / notes
  - similar to https://docs.stripe.com/api or https://docs.alchemy.com/reference/getnftsforowner-v3
  - this provides a more streamlined experience for the reader
  - it is future-proof -- more SDKs are easier to add later on
  - it also makes possibly non-uniform approaches in different SDKs more evident while writing docs

**Comment by oceans404** [comment]
Great suggestions! Thanks @a3s7p
